app/controllers/api/category_contents_controller.rb

```diff
@@ -0,0 +1,13 @@
+module API
+  class CategoryContentsController < APIController
+    def index
+      @primary_navigation, @secondary_navigation = Comfy::Cms::Category.navigation_categories
+      render json: @primary_navigation, scope: params[:locale].to_sym
```

Comment:
Is it a public-facing endpoint? If so, please consider white-listing available locales before typecasting user-provided value to `Symbol` objects.

As far as I know, symbols are not garbage collected in major ruby implementations, therefore it leaves space for an unlikely, but possible denial of service attack (assuming the rest of our infrastructure wouldn't prevent that). Some background: http://brakemanscanner.org/docs/warning_types/denial_of_service/